examples/websocket/Cargo.toml

```diff
@@ -11,6 +11,7 @@ path = "src/client.rs"
 [[bin]]
 name = "server"
 path = "src/server.rs"
+required-features = ["server"]
```

Comment:
Why?

Reply:
Ah, I was having trouble compiling (accidentally used `cargo make check`) so I was trying to figure things out. I must've committed this by accident.